guides/deployments/podman.md

```diff
@@ -127,79 +128,124 @@ Linux kernel doesn't support CVMs.
    ```
 
+   The example `DaemonSet` includes a `nodeSelector` that constrains the device
+   plugin to nodes with the `smarter-device-manager` label set to `enabled`.
+   Label the nodes that will include the FUSE device by using the following
+   command, or remove the `nodeSelector` from the manifest:
+
+   ```console
+   kubectl get nodes
+   kubectl label node <node name> smarter-device-manager=enabled
+   ```
+
 1. If you haven't already done so for your Coder deployment, enable workspace
    templates. To do so, go to **Manage** > **Admin** > **Templates**, and set
    the **Enable workspace templates** to **On**. Click **Save**.
 
-1. Create a YAML file with the following contents (the instructions ask the
-   cluster to request the FUSE device for each workspace):
+1. Create a
+   [workspace configuration file](../../workspaces/workspace-templates/templates.md)
+   that includes instructions for resource requests and resource limits (the
+   instructions ask the cluster to request the FUSE device for each workspace):
 
    ```yaml
    version: "0.2"
    workspace:
-       configure:
-           start:
-               policy: write
-       dev-urls:
-           policy: write
-       specs:
-           aws-ec2-docker:
-               container-image:
-                   policy: write
-               disk-size:
-                   policy: write
-               instance-type:
-                   policy: write
-           docker:
-               container-based-vm:
-                   policy: write
-               image:
-                   policy: write
-           kubernetes:
-               annotations:
-                   policy: read
-               container-based-vm:
-                   policy: write
-               cpu:
-                   policy: write
-               disk:
-                   policy: write
-               env:
-                   policy: write
-               gpu-count:
-                   policy: write
-               image:
-                   policy: write
-               labels:
-                   policy: read
-               memory:
-                   policy: write
-               node-selector:
-                   policy: read
-               privileged:
-                   policy: read
-               resource-requests:
-                policy: write
-                value:
-                   smarter-devices/fuse: "1"
-               resource-limits:
-                policy: write
-                value:
-                   smarter-devices/fuse: "1"
-               runtime-class-name:
-                   policy: read
-               tolerations:
-                   policy: read
+     specs:
+       kubernetes:
+         resource-requests:
+           policy: write
+           value:
+             smarter-devices/fuse: "1"
+         resource-limits:
+           policy: write
+           value:
+             smarter-devices/fuse: "1"
+   ```
+
+   A complete workspace template might look something like
```

Comment:
wdyt @jawnsy ?

Reply:
Awseome idea! :brain: 

Thanks Katie. Looks great to me